README.md

```diff
@@ -1,2 +1,9 @@
 # simple-chat
 A simple chat app using Angular and socket.io
+
+# License
+[MIT LICENSE](LICENSE)
```

Comment:
Add something like `This project is licensed under the terms of the [MIT License](LICENSE)`

Reply:
Done.